Tests/GitHubRepositories.tests.ps1

```diff
@@ -138,6 +147,193 @@ try
+        Context -Name 'For renaming a repository' -Fixture {
+            BeforeEach -Scriptblock {
+                $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid) -AutoInit
+                $suffixToAddToRepo = "_renamed"
+                $newRepoName = "$($repo.Name)$suffixToAddToRepo"
+            }
+            It "Should have the expected new repository name - by URI" {
+                $renamedRepo = $repo | Rename-GitHubRepository -NewName $newRepoName -Confirm:$false
+                $renamedRepo.Name | Should be $newRepoName
+            }
+
+            It "Should have the expected new repository name - by Elements" {
+                $renamedRepo = Rename-GitHubRepository -OwnerName $repo.owner.login -RepositoryName $repo.Name -NewName $newRepoName -Confirm:$false
+                $renamedRepo.Name | Should be $newRepoName
+            }
+            ## cleanup temp testing repository
+            AfterEach -Scriptblock {
+                ## variables from BeforeEach scriptblock are accessible here, but not variables from It scriptblocks, so need to make URI (instead of being able to use $renamedRepo variable from It scriptblock)
+                Remove-GitHubRepository -Uri "$($repo.svn_url)$suffixToAddToRepo"
+            }
+        }
+    }
+
+    Describe 'Updating repositories' {
+
+        Context -Name 'For creating a repository' -Fixture {
+            BeforeAll {
+                $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid) -Description $defaultRepoDesc -AutoInit
+            }
+            AfterAll {
+                Remove-GitHubRepository -Uri "$($repo.svn_url)"
+            }
+
+            It 'Should have the new updated description' {
+                $modifiedRepoDesc = $defaultRepoDesc + "_modified"
+                $updatedRepo = Update-GitHubRepository -OwnerName $repo.owner.login -RepositoryName $repo.Name -Description $modifiedRepoDesc
+                $updatedRepo.description | Should be $modifiedRepoDesc
+            }
+
+            It 'Should have the new updated homepage url' {
+                $updatedRepo = Update-GitHubRepository -OwnerName $repo.owner.login -RepositoryName $repo.Name -Homepage $defaultRepoHomePage
+                $repo.homepage | Should be $defaultRepoHomePage
+            }
+        }
+    }
+
+    Describe 'Get/set repository topic' {
+
+        Context -Name 'For creating and getting a repository topic' -Fixture {
+            BeforeAll {
+                $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid) -AutoInit
+            }
+            AfterAll {
+                Remove-GitHubRepository -Uri "$($repo.svn_url)"
+            }
+
+            It 'Should have the expected topic' {
+                $topic = Set-GitHubRepositoryTopic -OwnerName $repo.owner.login -RepositoryName $repo.Name -Name $defaultRepoTopic
+                $updatedRepo.names[0] | Should be $defaultRepoTopic
```

Comment:
Where is this `$updatedRepo` variable getting set?  Same question below.

Reply:
Silly mistake